src/wallet/asyncrpcoperation_shieldcoinbase.cpp

```diff
@@ -150,6 +156,21 @@ void AsyncRPCOperation_shieldcoinbase::main() {
     LogPrintf("%s",s);
 
     unlock_utxos(); // clean up
+
+    // !!! Payment disclosure START
```

Comment:
Code duplication that should later be moved to a method?

Reply:
This block of code is also used in `asyncrpcoperation_sendmany.cpp` and can be refactored at a later date.